doc/source/advanced.md

```diff
@@ -39,7 +39,45 @@ We recommend the community-maintained [nginx-ingress](https://github.com/helm/ch
 controller, [**kubernetes/ingress-nginx**](https://github.com/kubernetes/ingress-nginx).
 Note that Nginx maintains two additional ingress controllers.
 For most use cases, we recommend the community maintained **kubernetes/ingress-nginx** since that
-is the ingress controller that the development team has the most experience using.
+is the ingress controller that the development team has the most experience using. See below for
+**GCE Ingress** settings.
+
+#### GCE Ingress and manual HTTPS
+To set up HTTPS with GCE Ingress, you must:
+1. offload the proxy, enable `https` (and ideally, change it from 
+ external `LoadBalancer` to close it from outside world - we have ingress for that)
```

Comment:
You are suggesting a change from `LoadBalancer` to `NodePort`. Do we need `NodePort` over `ClusterIP`?

Yikes it becomes quite complicated to figure things out with so many parts in play.

Reply:
I believe it is: https://github.com/kubernetes/kubernetes/issues/26508#issuecomment-222376962.